crates/cli/src/op_angstrom.rs

```diff
@@ -167,3 +151,97 @@ async fn run_with_signer<S: AngstromMetaSigner>(
     .launch()
     .await
 }
+
+/// Validates that the sequencer URL is not a WebSocket URL.
+/// Returns an error if the URL scheme is "ws" or "wss".
+fn validate_sequencer_url(sequencer: &str) -> eyre::Result<()> {
+    let url = Url::parse(sequencer)?;
+    match url.scheme() {
+        "ws" | "wss" => Err(eyre::eyre!("Sequencer URL must be HTTP, not WS")),
+        _ => Ok(())
+    }
+}
+
+/// Add sequencer to normal nodes if it's not already in the list
+fn add_sequencer_to_normal_nodes(
+    angstrom_config: &mut AngstromConfig,
+    sequencer: &str
+) -> eyre::Result<()> {
```

Comment:
Would turn this into a method on `AngstromConfig`, just `add_sequencer`

Reply:
https://github.com/SorellaLabs/angstrom/pull/607/commits/c4c9edc3eb89765bc9d09ea983ce1e1a9b6b7013

Reply:
Moved `validate_sequencer_url()` as well. It's only used by OP Angstrom for now, but I think it belongs together with `add_sequencer()`
https://github.com/SorellaLabs/angstrom/pull/607/commits/dee6d1ac0278a48778c1be1a48b24d5f1b8135c9